serverless.yml

```diff
@@ -37,7 +37,14 @@ functions:
     handler: src/functions/getResult/framework/handler.handler
     events:
       - http:
-          path: test-result/{app-ref}/{staff-number}
+          path: test-result/{appRef}/{staffNumber}
+          method: get
+
+  getRegeneratedEmails:
+    handler: src/functions/getRegeneratedEmails/framework/handler.handler
+    events:
+      - http:
+          path: regeneratedemails/{appRef}
```

Comment:
As this is in the result service, the route resource will be test-results, so likely the path should be something like

`path: test-result/regenerated-email/{appRef}`

Reply:
agreed.  good shout